humam_tutorial.ipynb

```diff
@@ -112,8 +112,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# If using the Kernel EBRAINS-24.04, uncomment the following line to install the missing packages.\n",
-    "# !pip install dicthash nnmt xlrd"
+    "# If using the Kernel EBRAINS-24.04, install the following missing packages. Otherwise, comment the following line.\n",
+    "!pip install dicthash nnmt xlrd"
```

Comment:
The default should be the other way around: Assume the packages are installed, uncomment the following line. This allows to use the notebook in an automated fashion to test the environment for completeness. Same we do in the multi-area-model.
```suggestion
    "# If the environment does not already provide those packages, uncomment the following line.\n",
    "#!pip install dicthash nnmt xlrd"
```